### PR TITLE
Point brightness

### PR DESCRIPTION
**Pull request by pelikhan**
Dual of "plot brightness", reads the current brightness state of an LED.

### CODE REVIEW

**Comment by abchatra** [comment]
What rationale for the new block?

**Comment by pelikhan** [comment]
Symmetry. We have a block to set the LED brightness, but we don't have it to read it. We also support this for images.

**Comment by abchatra** [comment]
Symmetric to which block?
image

**Comment by pelikhan** [comment]
plot brightness

________________________________

File: libs/core/led.cpp [hunk 1]
```diff
@@ -62,18 +62,17 @@ namespace led {
...
      * @param x the horizontal coordinate of the LED
      * @param y the vertical coordinate of the LED
      */
-    //% help=led/point weight=76
-    //% blockId=device_point block="point|x %x|y %y"
+    //% help=led/point-brightness weight=76
+    //% blockId=device_point_brightness block="point|x %x|y %y brightness"
     //% parts="ledmatrix"
     //% x.min=0 x.max=4 y.min=0 y.max=4
     //% x.fieldOptions.precision=1 y.fieldOptions.precision=1
```
Comment (abchatra) [inline libs/core/led.cpp:73]: //% advanced=true

**Comment by abchatra** [comment]
Ok looks good. Other than it should be in more category right?

**Comment by pelikhan** [comment]
Moved to more.

**Comment by pelikhan** [comment]
image

**Pull request merged**